systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -893,6 +901,170 @@ void testConfigureDeploymentStrategy() {
         assertThat(kmm2.getSpec().getTemplate().getDeployment().getDeploymentStrategy(), is(DeploymentStrategy.ROLLING_UPDATE));
     }
 
+    @Test
+    void testRestoreOffsetsInConsumerGroup() {
+        final String SYNC_GROUP_OFFSETS_INTERVAL_SECONDS = "1";
+        final String topicSourceNameMirrored = "test-sync-offset-" + new Random().nextInt(Integer.MAX_VALUE);
+        final String topicTargetNameMirrored = kafkaClusterSourceName + "." + topicSourceNameMirrored;
+        final String consumerGroup = "mm2-test-consumer-group";
+        final String sourceProducerName = "mm2-producer-source-" + ClientUtils.generateRandomConsumerGroup();
+        final String sourceConsumerName = "mm2-consumer-source-" + ClientUtils.generateRandomConsumerGroup();
+        final String targetProducerName = "mm2-producer-target-" + ClientUtils.generateRandomConsumerGroup();
+        final String targetConsumerName = "mm2-consumer-target-" + ClientUtils.generateRandomConsumerGroup();
+
+        // Deploy source kafka
+        KafkaResource.kafkaClient().inNamespace(ResourceManager.kubeClient().getNamespace()).createOrReplace(
```

Comment:
You can do it by `KafkaResource.kafkaWithoutWait(...)` -> it's the same thing and we are using this in all test cases